src/App.js

```diff
@@ -4,9 +4,7 @@ import SinglePage from './components/singlePage/singlePage';
 
 function App() {
   return (
-    <div className="App">
       <SinglePage />
-    </div>
```

Comment:
css for background was on this class

Reply:
Was moved to singlePage.css, as App.js isn't used or called in the backend, `rubyEntryPoint.js` is used for that.